system-int-test/test/cmr/system_int_test/search/collection_cloud_hosted_search_test.clj

```diff
@@ -0,0 +1,49 @@
+(ns cmr.system-int-test.search.collection-cloud-hosted-search-test
+  "Integration tests for searching for records that are cloud hosted"
+  (:require
+   [clojure.test :refer :all]
+   [cmr.mock-echo.client.echo-util :as echo]
+   [cmr.indexer.data.concepts.tag :as itag]
+   [cmr.system-int-test.data2.core :as data2]
+   [cmr.system-int-test.data2.umm-spec-collection :as data-umm-c]
+   [cmr.system-int-test.system :as system]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.ingest-util :as ingest]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.system-int-test.utils.tag-util :as tags]))
+
+(use-fixtures :each (join-fixtures
+                      [(ingest/reset-fixture {"provguid1" "PROV1"})
+                       tags/grant-all-tag-fixture]))
+
+(defn- create-direct-dist-info
+  "Create a Direct Distribution"
+  []
+  {:DirectDistributionInformation {:Region "us-east-1"
+                                   :S3BucketAndObjectPrefixNames ["Bucket"]
```

Comment:
Remove :S3BucketAndObjectPrefixNames - it is optional in the schema and users may choose not to use this. But your code is dependent on it.

Actually create another test case - so that you have 2 test cases, one with S3BucketAndObjectPrefixNames and one without.

Reply:
does it matter to test with it? I don't think the code will care